desktop/cmp/error/ErrorMessage.ts

```diff
@@ -72,7 +72,7 @@ export const [ErrorMessage, errorMessage] = hoistCmp.withFactory<ErrorMessagePro
             ...rest
         } = props;
 
-        if (isNil(error)) return null;
+        if (isNil(error) && !message) return null;
```

Comment:
sorry -- isn't this intentional that you could have a fixed mesage, but only render when there was an error set?

It seems to me that whatever usage is requiring this is misuing the API